simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/StationSimulatorFunctionalTest.java

```diff
@@ -637,6 +636,55 @@ void shouldGetHeartbeatIntervalWithGetVariablesRequest() {
         });
     }
 
+    @Test
+    void shouldReplyToGetBaseReportRequest() {
+        stationSimulatorRunner.run();
+
+        GetBaseReportRequest request = new GetBaseReportRequest().withReportBase(FULL_INVENTORY);
+        String payload = new Call(UUID.randomUUID().toString(), ActionType.GET_BASE_REPORT, request).toJson();
+
+        stationSimulatorRunner.getStation(STATION_ID).sendMessage(new StationMessage(STATION_ID, StationMessage.Type.OCPP_MESSAGE, payload));
+
+        await().untilAsserted(() -> {
+            Optional<CallResult> responseFromStation = server.getReceivedCallResults(STATION_ID).stream().findAny();
+            assertThat(responseFromStation).isPresent();
+        });
+    }
+
+    @Test
+    void shouldSendNotifyReportOnGetBaseReportRequest() {
+        mockSuccessfulGetBaseReportAnswer();
+
+        stationSimulatorRunner.run();
+
+        GetBaseReportRequest request = new GetBaseReportRequest().withRequestId(200).withReportBase(FULL_INVENTORY);
+        String payload = new Call(UUID.randomUUID().toString(), ActionType.GET_BASE_REPORT, request).toJson();
+
+        stationSimulatorRunner.getStation(STATION_ID).sendMessage(new StationMessage(STATION_ID, StationMessage.Type.OCPP_MESSAGE, payload));
+
+        await().untilAsserted(() -> {
+            List<Call> stationCalls = server.getReceivedCalls(STATION_ID);
+
+            List<NotifyReportRequest> notifyReportRequests =
+                    stationCalls
+                            .stream()
+                            .filter(call -> call.getActionType() == ActionType.NOTIFY_REPORT)
+                            .map(call -> NotifyReportRequest.class.cast(call.getPayload()))
+                            .collect(toList());
+
+            sort(notifyReportRequests, new NotifyReportComparator());
+
+            int size = notifyReportRequests.size();
```

Comment:
`int size = notifyReportRequests.size() - 1;`
So that size - 1 can be omited below in 4 places

Reply:
done